.env

```diff
@@ -1,7 +1,7 @@
 SASS_PATH=./src/assets/styles
 REACT_APP_APPLICATION_NAME=$npm_package_name
 REACT_APP_OIDC_AUTHORITY="https://tunnistamo.test.hel.ninja/openid"
-REACT_APP_OIDC_CLIENT_ID="haitaton-admin-ui-dev"
+REACT_APP_OIDC_CLIENT_ID="haitaton-ui-dev"
```

Comment:
Was this suppossed to be in version control? I think it is good change because I use this same env in development and I don't see why there has been `admin` before.

Reply:
Yes, I thought that it would be good to finally make that change.